src/main/java/org/mockito/session/MockitoSessionLogger.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.session;
+
+import org.mockito.Incubating;
+import org.mockito.MockitoSession;
+
+/**
+ * Logger for hints emitted when finishing mocking for a {@link MockitoSession}.
+ *
+ * @since 2.13.4
+ */
+@Incubating
+public interface MockitoSessionLogger {
```

Comment:
Nice! Thank you for making this type public and placing it in the right package.